### ISSUE TITLE
Visualize already implemented instrument filters

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#1203]
Currently we filter instruments when the user comes from all details pages by the following attributes:
```
'survey': 'surveyIds',
'related-publication': 'relatedPublications.id',
'study': 'studyId'
```
We need to visualize these filter settings for the user. Ideally the user can edit these.

**Comment by rreitmann** [comment]
Implemented with #1199

**Comment by svietgen** [comment]
@rreitmann When I use Internet Explorer 11, I can choose a filter but this filter is not displayed afterwards.

![grafik](https://user-images.githubusercontent.com/22055122/29507829-1ecef38a-8652-11e7-82eb-a2aae94da791.png)
